gpoa/frontend/firefox_applier.py

```diff
@@ -115,24 +115,68 @@ def get_home_page(self):
             return homepage
         return None
 
-    def get_block_about_config(self):
+    def get_boolean_config(self, name):
         '''
-        Query BlockAboutConfig boolean property from the storage.
+        Query boolean property from the storage.
         '''
-        response = self.get_hklm_string_entry('BlockAboutConfig')
+        response = self.get_hklm_string_entry(name)
         if response:
-            if response.data.lower() in ['0', 'false', False, None, 'None']:
+            data = response.data if isinstance(response.data, int) else str(response.data).lower()
+            if data in ['0', 'false', None, 'none', 0]:
                 return False
-            return True
+            if data in ['1', 'true', 1]:
+                return True
 
         return None
 
+    def set_boolean_policy(self, name):
+        '''
+        Add boolean entry to policy set.
+        '''
+        obj = self.get_boolean_config(name)
+        if obj is not None:
```

Comment:
How about signalling that it is impossible to interpret the value of option?

Reply:
I think I would be nice in next PR. Let's merge that already works.